data/make_filelists.py

```diff
@@ -1333,6 +1333,211 @@ def get_v12v2_private():
                 ],
             },
         },
+        "2024": {
+            "HH": {
```

Comment:
Remove all the MC keys in 2024 (they refer to 2023BPix so they're not the right samples) and re-run it to fix the nanoindex